README.md

```
@@ -337,14 +375,25 @@ docker-compose up -d
 → trusta Realm 선택
 → Clients → trusta-gateway → Credentials 탭
 → Client Secret 값 확인
 ```
 
 ---
 
+### X-User-Enabled 누락으로 400 오류가 날 때
+
+Gateway 없이 단독 테스트 시 `X-User-Enabled` 헤더를 빠뜨리면 발생합니다.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)